File: backend/models/bslclass.js
module.exports = (sequelize, DataTypes) => {
  const BSLClass = sequelize.define('BSLClass', {
    class_number: {
      type: DataTypes.INTEGER,
      primaryKey: true,
      autoIncrement: false,
      allowNull: false,
    },
    description: {
      type: DataTypes.STRING,
      allowNull: true,
    },
    required_equipment: {
      type: DataTypes.JSONB,
      allowNull: false,
      defaultValue: {
        required: [],
        anyOf: [],
        optional: [],
      },
    },
  }, {
    tableName: 'bsl_classes',
    timestamps: false,
  });

  return BSLClass;
};

File: backend-python/app/models/bsl_class.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.orm import relationship

from app.database import Base


class BSLClass(Base):
    __tablename__ = "bsl_classes"

    class_number = Column(Integer, primary_key=True)
    description = Column(String)

    microbes = relationship(
        "Microbe",
        back_populates="bsl_class"
    )